app/client/views/shared/VueComponent/VueComponent.js

```diff
@@ -0,0 +1,6 @@
+Template.BlazeVueComponent.helpers({
```

Comment:
This may no longer be necessary, since we can use `VueComponent` along with mixins. Lets remove this layer of abstraction.